setup.py

```diff
@@ -11,44 +11,37 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import io
-import os
+
+# FIXME: Remove this setup.py completely.
```

Comment:
should we also have a TODO to remove release.py? I assume all uv-specific steps could be moved to a makefile script instead

Reply:
I assume release.py itself can be removed without major version update. I'm still modifying for it so still haven't ended with comment